transmart-copy/src/main/groovy/org/transmartproject/copy/table/Concepts.groovy

```diff
@@ -101,26 +106,49 @@ class Concepts {
         conceptCodeToConceptPath[conceptCode] = conceptPath
     }
 
+    void updateNameForConcept(
```

Comment:
Can this function be merged with `updatePathForConcept`?
The delete line can be simplified to `"delete from ${TABLE} where concept_cd = :conceptCode"`.